library/zlib/deflate/deflate_spec.rb

```diff
@@ -1,22 +1,24 @@
-# -*- encoding: us-ascii -*-
```

Comment:
Isn't the magic encoding comment enough for `parser` to figure out that literal strings are ASCII-8BIT, that is BINARY?